en/clubs/informationsorganet/body.md

```diff
@@ -1 +1,23 @@
-[I want access to Datasektionen's GitHub](https://dsekt.se/github-access)
\ No newline at end of file
+# IOR
+
+We are happy that you found IOR's page. We are also known as the _Donkeys_, which is a name that does not describe what we do very well. IOR is in charge of developing new systems and maintaining the Chapter’s current systems. In short, we are a group of students who enjoy programming and learning together. Everyone is welcome here, regardless of whether you have worked as a developer for a few years or are completely new to programming.
+
+## The Chapter’s systems
+
+The chapter’s systems are available on our [Github](https://dsekt.se/github-link-website), if you want access to see or edit something, you can request access through this [form] (https:// dsekt.se/ github-access).
+
+### Selection of systems
+
+Here is a list of some systems that previous students have built and that the chapter use extensively:
+
+- [Cashflow](https://cashflow.datasektionen.se) is probably our most used system. Here you can register expenses that you have made for the chapter and the Treasurer (or some other responsible person) will pay the money to you as soon as possible. Neat, right? [Mentometer](https://mentometer.datasektionen.se) is used to hold certain elections during chapter meetings (SM).
+- [aaallt](https://aaallt.datasektionen.se/) may not be our most exciting system, but here you can go to almost every other system the chapter has.
+
+### Current projects (updated November 2021)
+
+- Budget HD: Our current budget system has its problems and therefore the development of a new, better (hopefully) system has begun, in HD.
+- dUrn 2.0: Our current system for ballot-elections (see [the regulations](https://styrdokument.datasektionen.se/reglemente) 3.12.7 for more information) needs to be reworked a bit. The system is being rebuilt in Go and is intended to be ready for the elections at Glögg-SM 2021.
```

Comment:
Can we link directly to sections in the regulations? If not, that might be something to look into.

Reply:
Yes: `https://styrdokument.datasektionen.se/reglemente#3-12-7-urnval`

Reply:
Bar is in the way though